ui/lib/feed/src/components/tag-feed.tsx

```diff
@@ -42,8 +47,47 @@ const TagFeed = (props: TagFeedProps) => {
 
   const { t } = useTranslation('ui-lib-feed');
 
+  const { showNsfw } = useNsfwToggling();
+  const { data: loginData, loading: authenticating } = useGetLogin();
+  const isLoggedIn = !!loginData?.id;
+
+  let nsfwFilters;
+
+  /**
+   *  Check if the feed will be used inside the My Antenna page and
```

Comment:
why is the moderation applied only to the my-antenna feed? shouldn't it also be applied to the tag feed, the one that you can navigate to by clicking on a tag?

Reply:
I applied it to the my-antenna feed only because we decided to hide the NSFW beams when the NSFW setting is off on the My Antenna page, so I applied the filters to filter out the NSFW beams completely from the feed. However, on the tag feed page, we decided to display the nsfw beams, with an overlay, when the nsfw setting is off because we have that counter that displays how many beams there are for a specific tag, and if we hide the nsfw beams here, the number displayed won't match the number of beams a user sees on the page :)